composeApp/src/commonMain/kotlin/com/sangyoon/parkingpass/presentation/ui/ParkingLotDetailScreen.kt

```
@@ -362,3 +382,24 @@ private fun DashboardStatCard(
+        val parts = dateTime.split("T")
+        if (parts.size == 2) {
+            val timeParts = parts[1].split(":")
+            if (timeParts.size >= 2) {
+                "${timeParts[0]}:${timeParts[1]}"
+            } else {
+                "방금 전"
+            }
+        } else {
+            "방금 전"
+        }
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)